### PR TITLE
New deployment using Docker Swarm instead Docker Compose

### PR DESCRIPTION
**Pull request by AdrianSkierniewski**

### CODE REVIEW

File: ansible/templates/app-stack.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        constraints:
+          - node.role == manager
+      restart_policy:
+        condition: on-failure
+    ports:
+      - "80:80"
+      - "443:443"
+      - "8443:8443"
+    volumes:
+      - ./treafik.toml:/treafik.toml:ro
```
Comment (bartero) [inline ansible/templates/app-stack.yml]: `treafik` or `traefik`?
which name is correct???

File: ansible/templates/app-stack.yml [hunk 2]
```diff
@@ -0,0 +1,156 @@
+version: '3.6'
+services:
+
+  proxy:
+    image: traefik:1.6
+    command: -c /treafik.toml
```
Comment (bartero) [inline ansible/templates/app-stack.yml]: `treafik` or `traefik`?
which name is correct???

Comment (AdrianSkierniewski) [reply ansible/templates/app-stack.yml]: `traefik` is correct and yes I know it's difficult :)

**Review by PSkierniewski — APPROVED** [review]
:+1:

**Review by bartero — CHANGES_REQUESTED** [review]
`treafik` or `traefik`?
which name is correct???

**Review by bartero — APPROVED** [review]

**Pull request merged**